samples/sampler/stories/default/TooltipDecorator.js

```diff
@@ -1,30 +1,62 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {action} from '@enact/storybook-utils/addons/actions';
-import {boolean, select, text} from '@enact/storybook-utils/addons/knobs';
+import {boolean, number, select, text} from '@enact/storybook-utils/addons/knobs';
 import {storiesOf} from '@storybook/react';
-import Button, {ButtonBase} from '@enact/agate/Button';
-import TooltipDecorator from '@enact/agate/TooltipDecorator';
-import UiButton from '@enact/ui/Button';
+import Button from '@enact/agate/Button';
+import TooltipDecorator, {Tooltip, TooltipBase} from '@enact/agate/TooltipDecorator';
 import React from 'react';
 
+import iconNames from './icons';
+
 Button.displayName = 'TooltipDecorator';
-const Config = mergeComponentMetadata('TooltipDecorator', UiButton, ButtonBase, Button);
-const TooltipButton = TooltipDecorator({tooltipDestinationProp: 'decoration'}, Button);
+const Config = mergeComponentMetadata('TooltipDecorator',  TooltipDecorator, Tooltip, TooltipBase);
+const TooltipButton = TooltipDecorator(
+	{tooltipDestinationProp: 'decoration'},
+	Button
+);
+
+const prop = {
+	icons: ['', ...iconNames],
+	tooltipPosition: [
+		'above',
+		'above center',
+		'above left',
+		'above right',
+		'below',
+		'below center',
+		'below left',
+		'below right',
+		'left bottom',
+		'left middle',
+		'left top',
+		'right bottom',
+		'right middle',
+		'right top'
+	]
+};
 
 storiesOf('Agate', module)
 	.add(
 		'TooltipDecorator',
 		() => (
-			<TooltipButton
-				backgroundOpacity={select('backgroundOpacity', ['opaque', 'lightOpaque', 'transparent'], Config)}
-				disabled={boolean('disabled', Config)}
-				onClick={action('onClick')}
-				selected={boolean('selected', Config)}
-				size={select('size', ['smallest', 'small', 'large', 'huge'], Config)}
-				tooltipText={text('tooltip', Config, 'Tooltip')}
-			>
-				{text('children', Button, 'Click me')}
-			</TooltipButton>
+			<div style={{textAlign: 'center'}}>
+				<TooltipButton
+					backgroundOpacity={select('backgroundOpacity', ['opaque', 'lightOpaque', 'transparent'], Config)}
```

Comment:
This prop doesn't work from before. Coul you make a Jira issue and proceed to the next ?

Reply:
I just tested locally and it seems this does work. I will attach some screenshots below. Would you like to remove this knob since neither Sandstone has it?
![lightOpaqueTooltip](https://user-images.githubusercontent.com/63335823/106448422-6bfe6d00-648b-11eb-8ea6-0966ae2e5dcd.png)
![tooltipOpaque](https://user-images.githubusercontent.com/63335823/106448425-6c970380-648b-11eb-8a51-c66bb588ebe9.png)
![transparentTooltip](https://user-images.githubusercontent.com/63335823/106448428-6d2f9a00-648b-11eb-9b90-aaa2d7588cf9.png)